plugins/aws/bedrock/customModelEncryptionEnabled.js

```diff
@@ -0,0 +1,72 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'Custom Model Encryption Enabled',
+    category: 'BedRock',
+    domain: 'Application Integration',
+    description: 'Ensure that an Amazon Bedrock custom model is encrypted by AWS KMS Customer Master Keys.',
+    more_info: 'When you encrypt AWS Bedrock custom model using your own AWS KMS Customer Master Keys (CMKs) for enhanced protection, you have full control over who can use the encryption keys to access your custom model.',
+    recommended_action: 'Encrypt Bedrock custom model using AWS KMS Customer Master Keys',
+    link: 'https://docs.aws.amazon.com/bedrock/latest/userguide/encryption-custom-job.html',
+    apis: ['Bedrock:listCustomModels', 'Bedrock:getCustomModel'],
+
+    run: function(cache, settings, callback) {
+        var results = [];
+        var source = {};
+        var regions = helpers.regions(settings);
+
+        async.each(regions.bedrock, function(region, rcb){
+            var listCustomModels = helpers.addSource(cache, source,
+                ['bedrock', 'listCustomModels', region]);
+
+            if (!listCustomModels) return rcb();
+
+            if (listCustomModels.err && listCustomModels.err.message.includes('This service may not be available in')) {
```

Comment:
```suggestion
            if (listCustomModels.err && listCustomModels.err.message.includes('This service may not be available in')) {
```

can we only add regions that have bedrock service availabe in bedrock.regions?